### ISSUE TITLE
[Duplicate-Detection]  Unbekannt / Unbekannt should not be taken into account when searching for duplicates

### ISSUE THREAD

**Issue by markusmann-vg** [SORMAS-Foundation/SORMAS-Project#4769]
### Situation Description
Currently the "Unbekannt / Unbekannt" persons/cases are take into account when searching for duplicates. 
We should ignore those cases/persons

**Comment by MateStrysewske** [comment]
@markusmann-vg This directly contradicts #4533 which has just been developed. What's the reason for this request?

**Comment by JaquM** [comment]
@MateStrysewske 
In case of non-named reports, no person is created in SurvNet. During an initial import, the name UNKNOWN UNKNOWN is automatically set. Merging is not possible, so every single unnamed case will be displayed with every other unknown case in the duplicate search. Therefore these should be taken out of the duplicate search completely.

**Comment by bernardsilenou** [comment]
@JaquM 
The value UNKNOWN  can be as a category to many variables. Issue #4533 is about UNKNOWN  sex but your explanation seem to be about UNKNOWN  as a name, is this right?
In the duplicate person detection, UNKNOWN  as a name is independent of UNKNOWN as a category of sex. I am lost somehow

**Comment by JaquM** [comment]
@bernardsilenou 
Yes, it is just about Unknown for name and last name in this part.

**Comment by MateStrysewske** [comment]
@JaquM @bernardsilenou @markusmann-vg I feel like this is a very bad idea for two reasons:

a) This is VERY hacky and can therefore easily cause problems (and is a weird special case that nobody who does any development work for this project will think about)
a) We have no idea whether the name "Unknown" - or "Unbekannt" - might be an actual name somewhere in the world; if that's the case, this "feature" would break legitimate duplicate detections for such persons

If it's necessary that cases with "UNKNOWN" persons are added to SORMAS via SurvNet, I think this needs to be handled in the converter (e.g. by using a random combination of numbers or something like that).

**Comment by bernardsilenou** [comment]
@MateStrysewske I absolutely agree with you. It is very easy to find an African name like `Nkwounn` whick would be excluded also since the string similarity method may be independent of the character order.
